lang/java/reef-tang/tang-test-jarA/src/main/java/org/apache/reef/tang/examples/A.java

```diff
@@ -19,5 +19,4 @@
 package org.apache.reef.tang.examples;
 
 public class A {
-  private int iAmAClass = 0;
```

Comment:
I don't know why these tests were disabled. Possibly because of build order: `TestClassLoaders` is part of Tang tests, and is executed before tang-test-jar* are built, thus the tests fail when executed as part of `mvn clean install`. If complete build is run first, and tests second, these tests pass (after editing jar version). Thus, it probably makes sense to put some effort into changing order of build and test and to re-enable these tests.

However, the change in this PR doesn't suggest to remove `TestClassLoaders` and tang-test-jar* projects. The only thing this PR removes is a field which is never used in these tests. Unless there is a very clear need to use this field in a test in a meaningful way (not just to prevent it from being unused), I don't think we should keep it. The tests seem to need only the structure of the classes and constructors, not any fields in them.